app/configurator/components/chart-configurator.tsx

```diff
@@ -120,6 +118,33 @@ const DataFilterSelectGeneric = ({
   );
 };
 
+const MoveAndDragButtons = ({
+  onClickUp,
+  onClickDown,
+  className,
+}: {
+  onClickUp: () => void;
+  onClickDown: () => void;
+  className?: string;
+}) => {
+  return (
+    <>
+      <Button className={className} variant="arrow" onClick={onClickUp}>
+        ▲
```

Comment:
This probably doesn't matter at all, but shouldn't we use <Icon name="caretUp" />?

Reply:
Nevermind, I see that you changed it in following commits :)